src/ucp/core/ucp_worker.c

```diff
@@ -2532,27 +2532,27 @@ void ucp_worker_destroy(ucp_worker_h worker)
 
     UCS_ASYNC_BLOCK(&worker->async);
     uct_worker_progress_unregister_safe(worker->uct, &worker->keepalive.cb_id);
+    ucp_worker_discard_uct_ep_cleanup(worker);
     ucp_worker_destroy_eps(worker, &worker->all_eps, "all");
     ucp_worker_destroy_eps(worker, &worker->internal_eps, "internal");
     ucp_am_cleanup(worker);
-    ucp_worker_discard_uct_ep_cleanup(worker);
     /* Put ucp_worker_remove_am_handlers after ucp_worker_discard_uct_ep_cleanup
      * to make sure iface->am[] always cleared.
      * ucp_worker_discard_uct_ep_cleanup might trigger ucp_worker_iface_deactivate
      * which further set iface->am[UCP_AM_ID_WIREUP].
      */
     ucp_worker_remove_am_handlers(worker);
 
-    if (worker->flush_ops_count != 0) {
-        ucs_warn("worker %p: %u pending operations were not flushed", worker,
-                 worker->flush_ops_count);
-    }
-
     if (worker->num_all_eps != 0) {
         ucs_warn("worker %p: %u endpoints were not destroyed", worker,
                  worker->num_all_eps);
     }
 
+    if (worker->flush_ops_count != 0) {
+        ucs_warn("worker %p: %u pending operations were not flushed", worker,
+                 worker->flush_ops_count);
+    }
```

Comment:
why needed to move? IMO warning order should not matter

Reply:
it is leftover from the previous solution

Reply:
removed